sources/packages/backend/apps/api/src/app.aest.module.ts

```diff
@@ -57,11 +55,16 @@ import {
   EducationProgramOfferingControllerService,
   NoteAESTController,
   RestrictionAESTController,
+  InstitutionUserControllerService,
+  ApplicationControllerService,
 } from "./route-controllers";
 import { AuthModule } from "./auth/auth.module";
-import { ApplicationControllerService } from "./route-controllers/application/application.controller.service";
-import { InstitutionUserControllerService } from "./route-controllers/institution-user/institution-user.controller.service";
-import { WorkflowClientService, SequenceControlService } from "@sims/services";
+import {
+  SequenceControlService,
+  StudentRestrictionsService,
+  WorkflowClientService,
+} from "@sims/services";
+import { MSFAANumberService, ReportService } from "@sims/integrations/services";
```

Comment:
Do we need to move the MSFAAService with all the methods to integrations lib?

All the methods in this service are not used by the integration I believe. 

For e.g `getCurrentValidMSFAANumber`, `isMSFAANumberValid` and `createMSFAANumber` are used inside applications service and I believe not anywhere else. 

Hence we can have a MSFAA number service at API scope with these methods used by API and one in libs/integration with the methods used in integration.

Reply:
this class from integration has methods only needed for integration 
![image](https://user-images.githubusercontent.com/77353155/210413119-78ac27e7-b0c2-434e-bc01-8f610062a7c6.png)
![image](https://user-images.githubusercontent.com/77353155/210413134-ddade0a8-858f-452a-b9d7-d8975b169b59.png)
the methods you mentioned are not part of the integration class.

these classes from integration are not needed here. refactored